cms/static/js/spec/views/pages/group_configurations_spec.js

```diff
@@ -74,7 +74,7 @@ define([
                     view = renderPage(),
```

Comment:
You should add a test point that the "you have unsaved changes" messages is shown if there is an unsaved content group.